### PR TITLE
Feature/oven

### PR DESCRIPTION
**Pull request by Feetjexx**
Added new oven mesh to `oven`  folder and into the oven prefab

### CODE REVIEW

**Review by TryFox — COMMENTED** [review]
I think the scale of the oven is a bit to small, please look into that
Also please say what you changed in the summary, not in the description

File: Assets/Scenes/Develop.unity
Comment (TryFox) [inline Assets/Scenes/Develop.unity:1]: Don't push develop changes

**Review by TryFox — CHANGES_REQUESTED** [review]
Never push develop changes unless you have to change something in the settings

**Review by TryFox — COMMENTED** [review]
Position issue has been fixed, still have to fix the scaling issue

**Review by TryFox — APPROVED** [review]

File: Assets/Prefabs/Train/TrainController/Train.prefab
Comment (TryFox) [inline Assets/Prefabs/Train/TrainController/Train.prefab:1]: look into if this change is needed

Comment (lucasvdiepen) [reply Assets/Prefabs/Train/TrainController/Train.prefab:1]: Change is for the position of the oven in the train, so this is correct.

**Review by lucasvdiepen — APPROVED** [review]

**Pull request merged**